edbdeploy/data/virtualbox/playbook.yml

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
Do we need a specific ansible playbook for virtualbox? Please see https://github.com/EnterpriseDB/postgres-deployment/tree/master/edbdeploy/data/ansible

Reply:
I just used the same playbook that was used for vmware-wkstn

Reply:
Please use the `EDB-RA-1.yml`, `EDB-RA-2.yml` and `EDB-RA-3.yml` playbook files, located in `data/ansible`.

Reply:
Okay